src/drawer.test.miscellaneous.ts

```diff
@@ -0,0 +1,274 @@
+import { html } from 'lit';
+import { expect, test } from './playwright/test.js';
+
+test('defines itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(
+    () => html`<glide-core-drawer label="Label">Content</glide-core-drawer>`,
+  );
+
+  const host = page.locator('glide-core-drawer');
+
+  await expect(host).toBeDefined('glide-core-drawer');
+});
+
+test(
+  'can be opened',
+  { tag: '@miscellaneous' },
+  async ({ browserName, mount, page, setProperty }) => {
+    test.skip(
+      browserName === 'webkit',
+      '"toggle" is dispatched inconsistently in CI. Or perhaps on Ubuntu. Try removing this when Webkit 26 is stable (TODO).',
+    );
```

Comment:
Running these tests locally on a Mac with:

```bash
PLAYWRIGHT_BROWSER=webkit pnpm exec playwright test --config ./src/playwright/playwright.config.ts --project functionality
```

pass without an issue. But if you look back at the previous runs on CI, you'll notice these tests flaking out at times. Better to skip these for now and revisit soon™.